flask_backend/app.py

```diff
@@ -11,7 +11,7 @@
 app = Flask(__name__)
 CORS(app)
 
-#app
+#apps
```

Comment:
jobs and job_list LGTM

Reply:
getJobsFromQuery function looks good. May need to do some more testing on request.get once we have the frontend linked.

Reply:
Yes, I was planning on doing that. Thanks for the reminder though